app/react/src/server/cra_config.js

```diff
@@ -0,0 +1,83 @@
+import semver from 'semver';
+
+let MiniCssExtractPlugin;
+let normalizeCondition;
+
+export function isReactScriptsInstalled() {
+  try {
+    // eslint-disable-next-line global-require, import/no-extraneous-dependencies
+    const reactScriptsJson = require('react-scripts/package.json');
+    if (semver.lt(reactScriptsJson.version, '2.0.0')) return false;
+
+    // eslint-disable-next-line global-require, import/no-extraneous-dependencies, prefer-destructuring
+    normalizeCondition = require('webpack/lib/RuleSet').normalizeCondition;
+    // eslint-disable-next-line global-require, import/no-extraneous-dependencies
+    MiniCssExtractPlugin = require('mini-css-extract-plugin');
```

Comment:
Same here. Do we need it as a dep?

Reply:
We don't need this as a dependency since we're first checking if `react-scripts` is installed. If so, then `mini-css-extract-plugin` will be installed.

Reply:
So you can require it already in the `applyCRAWebpackConfig` func and remove the state from the module (IMO Node caches imported modules, so you don't need to care about this issue).